desktop/webpack.config.js

```diff
@@ -1,6 +1,3 @@
-/**
```

Comment:
note that eslint and the import order script didn't seem to work in this file

Reply:
Looks like `bin/import-order.sh` is looking for `External dependencies`, with lowercase 'd'